conjur/api/client.py

```diff
@@ -9,19 +9,17 @@
 
 # Builtins
 import logging
-
-# Internals
 import netrc
 
-from conjur.errors import InvalidOperationException
+# Internals
+from conjur.errors import InvalidCertificateVerificationException
```

Comment:
InvalidCertificateVerificationException -> CertificateVerificationException
one can think of invalid to be invalid certificate